shopfloor_single_product_transfer/services/single_product_transfer.py

```diff
@@ -356,11 +401,14 @@ def _create_move_from_location(
             "product_id": product.id,
             "product_uom": product.uom_id.id,
             "product_uom_qty": quantity,
-            "location_id": location.id,
             "location_dest_id": picking_type.default_location_dest_id.id,
             "origin": self.work.menu.name,
             "picking_type_id": picking_type.id,
         }
+        if location:
+            move_vals["location_id"] = location.id
+        if package:
+            move_vals["package_id"] = package.id
```

Comment:
This field does not exist. You need to leverage forced_package_id in the action_assign().
I would create a package_level and set it on the move after action_confirm. This way the move has a picking_id you can use on the package level creation.